schema/release-schema.json

```
@@ -443,6 +443,11 @@
           "description": "The period for decision making regarding the contract award. The end date should be the date on which an award decision is due to be finalized. The start date may be used to indicate the start of an evaluation period.",
           "$ref": "#/definitions/Period"
         },
+        "standstillPeriod": {
+          "title": "Standstill period",
+          "description": "The period starting from the dispatch of the notice of intention to award, during which suppliers can challenge the award decision.",
```

This had put the essential semantics in the second clause.

"notice of intention to award" is not used in OCDS. We use "award notice": https://github.com/open-contracting/standard/blob/1.2-dev/schema/codelists/documentType.csv If desired we can add "This notice is called a notice of intention to award." to the awardNotice code description.

We also use "publication" not "dispatch".

Per https://github.com/open-contracting/standard-development-handbook/issues/249 and #1416, "suppliers" is typically reserved for the actual suppliers (winners).

@JachymHercher What is best here: "unsuccessful bidders", "unsuccessful tenderers" or "potential suppliers"? As I understand, only participants (bidders/tenderers) can challenge.

I added "before the contract signature" to make explicit that the buyer or procuring entity cannot sign the contract during this period.

```suggestion
          "description": "The period during which unsuccessful bidders can challenge the award decision before the contract signature, starting from the publication of the award notice.",
```

---

Hmm my understanding is that the notice of intention award is only sent to the unsuccessful bidders and then the award notice (in OCDS terms) is published after the contract is actually awarded, i.e. after the standstill period. At least, that seems to be the case in the UK.

---

I would say "during which the award decision can be challenged". In the EU, awards can be challenged by potential suppliers as well as, sometimes, e.g. anti-corruption NGOs. 

(There is some legal magic involved - to demonstrate legal standing, you need to show you fall under Art. 1(3) of the [remedies Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:01989L0665-20140417):) 

> "3.  Member States shall ensure that the review procedures are available, under detailed rules which the Member States may establish, at least to any person having or having had an interest in obtaining a particular contract and who has been or risks being harmed by an alleged infringement."

---

@JachymHercher And for "starting from the publication of ..." would you use "award notice" (as defined in documentType.csv), "notice of intention to award" (not defined anywhere currently), or something else?

---

Not (or not just) "award notice", as at least in the EU it is not about a "notice" per se, see https://github.com/open-contracting/standard/issues/1206#issuecomment-917008173. 

The relevant provision, at least for the EU, is [Art. 2a of the Remedies Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:01992L0013-20140417&from=EN): 

> A contract may not be concluded [...] at least 10 calendar days with effect from the day following the date on which the **contract award decision is sent to the tenderers and candidates concerned** 

I see two options:
1) Describe it generally, without a reference to any concrete document. E.g. "starting from the moment they are informed about the award". 
2) Introduce a new document (e.g. "Intention to award letter" mentioned by Duncan in https://github.com/open-contracting/standard/issues/1142#issuecomment-904182601), add it to DocumentType.csv and use it in `tender.standstillPeriod`, for example as "starting from the receipt of the intention to award letter or the publication of the award notice." I would not use the word "notice" for this new document, as I like to think of notices as something that is fully public.

I would prefer 1), as it is both simpler and more general.

___ 

Also, "unsuccessful tenderers", not "unsuccessful bidders", per `organizationRole`.

---

Thanks, both. Both UNCITRAL and the World Bank use the word "notice" to describe the letter of intention to award, but since it isn't a public document, I agree that it's better to go with 1 than to add it to `DocumentType.csv`.

For the field description, how about:

> The period during which the award decision can be challenged before the contract signature, starting from the date that tenderers are informed about the award decision.

---

Sounds good!

---

@jpmckinney are you happy with the suggestion in https://github.com/open-contracting/standard/pull/1421#discussion_r746184334?

---

👍 to using the passive tense to avoid specifying who can challenge and to rewording the "starting from" clause.

"before the contract signature" modifies "period" but it's quite far from it. In terms of start date, some jurisdictions start the clock when informed, others when the decision was made (though usually this is treated as the same). So:

> The period before the contract signature during which the award decision can be challenged. This period typically starts on the date on which the tenderers are informed about the award decision.

---

Updated in https://github.com/open-contracting/standard/pull/1421/commits/566384d8330aedc26cbe1d6eef77278702bf0315